File: SIT-Test_Solution/20-12/Python3/Baekjoon01977_PerfectSquare.py
# Baekjoon01977_PerfectSquare.py
m = int( input() )
n = int( input() )

sumSquare, count, minVal = 0, 0, 10001

for i in range( 1, 101 ):
	square = i * i
	if ( square > n ):
		run = False
	else:
		if ( m <= square ):
			count += 1
			sumSquare += square
			minVal = min( minVal, square )

if ( count ):
	print( sumSquare )
	print( minVal )
else:
	print( -1 )

File: SIT-Test_Solution/20-12/C/Baekjoon01977_PerfectSquare.c
// Baekjoon01977_PerfectSquare.c
// https://www.acmicpc.net/problem/1977
#include <stdio.h>

int main(void) {
	int n, m;
	scanf("%d %d", &n, &m);

	int sum = 0, minVal = 0, num = 0;
	for ( int i = 1; i <= 100; i++ ) {
		int square = i * i;
		if ( square > m )
			break;
		else if ( square >= n ) {
			if ( !minVal )
				minVal = square;
			sum += square;
			num++;
		}
	}

	if ( num )
		printf("%d\n%d\n", sum, minVal);
	else
		printf("%d\n", -1);

	return 0;
}